pyracing/response_objects/session_data.py

```diff
@@ -0,0 +1,285 @@
+                self.bestquallapat = dict['bestquallapat']
+                self.car_color2 = dict['car_color2']
+                self.suit_color2 = dict['suit_color2']
+                self.clubname = dict['clubname']
+                self.suit_color1 = dict['suit_color1']
+                self.aggchamppoints = dict['aggchamppoints']
+                self.suit_color3 = dict['suit_color3']
+                self.custid = dict['custid']
+                self.interval = dict['interval']
+                self.oldlicenselevel = dict['oldlicenselevel']
+                self.avglap = dict['avglap']
+                self.classinterval = dict['classinterval']
+                self.simsesnum = dict['simsesnum']
+                self.quallaptime = dict['quallaptime']
+                self.track_category = dict['track_category']
+                self.carclassid = dict['carclassid']
+                self.newttrating = dict['newttrating']
+                self.restrictresults = dict['restrictresults']
+                self.clubid = dict['clubid']
+                self.carnum = dict['carnum']
+                self.startpos = dict['startpos']
+                self.carid = dict['carid']
+                self.lapscomplete = dict['lapscomplete']
+                self.car_number_color1 = dict['car_number_color1']
+                self.car_number_color2 = dict['car_number_color2']
+                self.licensegroup = dict['licensegroup']
+                self.car_number_color3 = dict['car_number_color3']
+                self.pos = dict['pos']
+                self.bestlaptime = dict['bestlaptime']
+                self.incidents = dict['incidents']
+                self.champpoints = dict['champpoints']
+                self.heatinfoid = dict['heatinfoid']
+                self.simsestypename = dict['simsestypename']
+                self.finishpos = dict['finishpos']
+                self.license_change_oval = dict['license_change_oval']
+                self.carsponsor2 = dict['carsponsor2']
+                self.helm_pattern = dict['helm_pattern']
+                self.carsponsor1 = dict['carsponsor1']
+                self.divisionname = dict['divisionname']
+                self.carnumberslant = dict['carnumberslant']
+                self.reasonoutid = dict['reasonoutid']
+                self.finishposinclass = dict['finishposinclass']
+                self.optlapscomplete = dict['optlapscomplete']
+                self.lapslead = dict['lapslead']
+                self.oldsublevel = dict['oldsublevel']
+                self.displayname = dict['displayname']
+                self.league_points = dict['league_points']
+                self.evttypename = dict['evttypename']
+                self.clubshortname = dict['clubshortname']
+                self.bestquallapnum = dict['bestquallapnum']
+                self.newlicenselevel = dict['newlicenselevel']
+
+        self.driver = [Driver(x) for x in dict['rows']]
+
+
+class RaceLapsAll:
```

Comment:
This class name doesn't really tell me what this is, is it `AllRaceLaps` or `AllRaceLapsFromSession` or something like that?

Reply:
I'm not sure of a better naming convention. I can add some descriptions in the modules, but since it's a class in the module 'session_data', it pertains to a single session. 

RaceLapsAll is all laps from the race. Literally all of them them from every car. If a race has 10 laps with 10 drivers, it will return 100 laps (assuming they all finish)